src/components/10-atoms/input-file/index.scss

```diff
@@ -57,11 +53,11 @@
 
 .a-input-file--large {
   min-height: 40px;
-  @include _text(default);
+  @include text(t4);
 
   @include breakpoint($mediaquery-xs-up) {
     min-height: 50px;
-    @include _text(medium)
+    @include text(t3);
```

Comment:
At this component is default -> t4 and medium -> t3. Thats different to button.

Reply:
Input file `large` is broken #1838. I copied the text sizes from button now.